### PR TITLE
mrboom 5.2

### PR DESCRIPTION
**Pull request by chenrui333**
[`action-homebrew-bump-formula`](https://github.com/dawidd6/action-homebrew-bump-formula)

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/mrboom.rb --verbose
Error: mrboom: failed
An exception occurred within a child process:
  NoMethodError: undefined method `[]' for nil:NilClass
```

File: Formula/mrboom.rb [hunk 1]
```diff
@@ -24,21 +24,8 @@ def install
...
-        Timeout.timeout(10) do
-          Process.wait pid
-        end
-      rescue Timeout::Error
-        Process.kill "KILL", pid
-      end
-    end
+    # gui app
+    assert_match "Set joysticks dead zone, default is 8000",
+      shell_output("#{bin}/mrboom --help")
```
Comment (SMillerDev) [inline Formula/mrboom.rb:29]: Why are we removing a test in favor of a help text?

Comment (chenrui333) [reply Formula/mrboom.rb:29]: I think that is a GUI app, that is why I updated the test back to a simpler version.

Comment (SMillerDev) [reply Formula/mrboom.rb:29]: Did the old test not test the app? Because otherwise I think that one should be fixed instead.

Comment (chenrui333) [reply Formula/mrboom.rb:29]: with mrboom 5.2, let me see if the old test works fine or not.

**Comment by chenrui333** [comment]
same error as before now

```
==> Testing mrboom
/usr/bin/sandbox-exec -f /private/tmp/homebrew20210117-22678-130u5d9.sb ruby -W0 -I $LOAD_PATH -- /usr/local/Homebrew/Library/Homebrew/test.rb /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/mrboom.rb --verbose
An exception occurred within a child process:
Error: mrboom: failed
  NoMethodError: undefined method `[]' for nil:NilClass
```

**Pull request closed without merging**